### PR TITLE
Feature merge iso traces

### PR DESCRIPTION
**Pull request by poshul**
Added utility function MSChromatogram to allow merging the ChromatogramPeaks from two MSChromatograms. Function is useful for summing intensity of multiple isotope traces in an isotopic envelope.

### CODE REVIEW

File: src/openms/source/FORMAT/HANDLERS/MzMLSpectrumDecoder.cpp [hunk 1]
```diff
@@ -306,6 +306,7 @@ namespace OpenMS
 
     if (data.size() > 2) 
     {
+      spectrum.sortByPosition();
```
Comment (timosachsenberg) [inline src/openms/source/FORMAT/HANDLERS/MzMLSpectrumDecoder.cpp:309]: is this correct regarding potential data arrays? is this from the other PR?

Comment (poshul) [reply src/openms/source/FORMAT/HANDLERS/MzMLSpectrumDecoder.cpp:309]: This is vestigial from another branch. I'm gonna work to disentangle this one.

File: src/topp/FileFilter.cpp [hunk 2]
```diff
@@ -110,7 +110,7 @@ Depending on the input file type, additional specific operations are possible:
 
 The priority of the id-flags is (decreasing order): remove_annotated_features / remove_unannotated_features -> remove_clashes -> keep_best_score_id -> sequences_whitelist / accessions_whitelist
 
-MS2 and higher spectra can be filtered according to precursor m/z (see 'peak_options:pc_mz_range'). This flag can be combined with 'rt' range to filter precursors by RT and m/z.
+MS2 and higher spectra can be filtered according to precursor m/z (see 'peak_options:pc_mz_range'). This flag can be combined with 'rt' range to filter precurso_rs by RT and m/z.
```
Comment (timosachsenberg) [inline src/topp/FileFilter.cpp:113]: _rs?

**Comment by poshul** [comment]
Closing PR to reopen with correct commits.

**Pull request closed without merging**